Camera:
  Focal: 13.971313932573906
  High: 650.0
Center:
  Camera_high: 620
  Center_X: 320
  Center_Y: 260
  Front: 91
  Horizon: 134
  Inner: 37
  Outer: 224
  X: 308
  Y: 266
FPS: 16
HSV:
  Ball:
  - 0
  - 26
  - 30
  - 100
  - 18
  - 100
  Blue:
  - 217
  - 261
  - 74
  - 100
  - 29
  - 100
  Green:
  - 150
  - 213
  - 51
  - 96
  - 17
  - 98
  White:
  - 197
  - 168
  - 0
  - 82
  - 0
  - 18
  Yellow:
  - 39
  - 79
  - 10
  - 100
  - 56
  - 100
  black:
    angle: 1
    gray: 24
  white:
    angle: 3
    gray: 193
Parameterbutton: 4
SCAN:
  Angle_Near_Gap: 4
  Angle_range_1: 2
  Angle_range_2_3: 3
  Dont_Search_Angle_1: 30
  Dont_Search_Angle_2: 149
  Dont_Search_Angle_3: 275
  Magn_Far_End: 224
  Magn_Far_Start: 154
  Magn_Middle_Start: 50
  Magn_Near_Gap: 4
  Magn_Near_Start: 35
blackItem:
  angle: 3
  gray: 40
whiteline:
  angle: 4
  gray: 100
